monitoring/rid_qualifier/host/config.py

```diff
@@ -10,22 +10,21 @@
 
 # These keys map to entries in the Config class
 KEY_AUTH_SPEC = 'AUTH_SPEC'
+ENV_KEY_REDIS_URL = os.environ['REDIS_URL']
+ENV_KEY_RID_HOST_URL = os.environ['RID_HOST_URL']
+ENV_KEY_PORT = os.environ['PORT']
 
 
 workspace_path = os.path.join(os.path.abspath(os.path.dirname(__file__)), 'workspace')
 os.environ["OAUTHLIB_INSECURE_TRANSPORT"] = "1"
 
 
 class Config(object):
-  AUTH_SPEC = os.environ[ENV_KEY_AUTH]
-  REDIS_URL = os.environ['REDIS_URL']
```

Comment:
We still want these variables defined down here in the `Config`, we just want to use, e.g., `ENV_KEY_REDIS_URL` rather than the magic string `'REDIS_URL'`.